alpha/apps/kaltura/modules/kmcng/actions/kmcngAction.class.php

```diff
@@ -230,6 +230,12 @@ private function initConfig($deployUrl, $kmcngParams, $enforceSecureProtocol, $r
                     "uri" => '/apps/kmcng/' . $kmcngParams["kmcng_version"] . "/public/playerWrapper.html"
                 );
 
+		$mediaRepurposingUrl = null;
```

Comment:
since we dont have any addition to the url why not just add it in one liner like analyticsServer and kpfServer